docs/tutorials/deployments.md

```diff
@@ -412,17 +412,44 @@ INFO:     Uvicorn running on http://127.0.0.1:4200 (Press CTRL+C to quit)
 ```
 </div>
 
-Note the message to set `PREFECT_API_URL` so that you're coordinating flows with this API instance.
+!!! note "Set the `PREFECT_API_URL` for your server"
+    Note the message to set `PREFECT_API_URL` &mdash; configuring the URL of your Prefect Orion server or Prefect Cloud makes sure that you're coordinating flows with the correct API instance.
 
-Go back to your first terminal session and run this command to set the API URL:
+    Go to your first terminal session and run this command to set the API URL to point at the Prefect Orion instance just started:
+
+    <div class='terminal'>
```

Comment:
This all looks good. Copying the code snippet doesn't work well because it contains a lot more than just the code, so we could separate it. Your call.

Reply:
Gets too busy if we separate everything out. Contextually this is covered in more detail elsewhere